packages/calling-component-bindings/src/handlers/createHandlers.ts

```diff
@@ -61,6 +63,8 @@ export type CallingHandlers = {
   onRemoveParticipant: (userId: string) => Promise<void>;
   onDisposeRemoteStreamView: (userId: string) => Promise<void>;
   onDisposeLocalStreamView: () => Promise<void>;
+  /* @conditional-compile-remove(dialpad) */
+  onSendDtmfTone?: (dtmfTones: DtmfTone) => Promise<void>;
```

Comment:
argument should be non-plural also
```suggestion
  onSendDtmfTone?: (dtmfTone: DtmfTone) => Promise<void>;
```

Reply:
fixed that!